docs/content/release-process.md

```diff
@@ -8,16 +8,21 @@ url = "https://github.com/fermyon/spin/blob/main/docs/content/release-process.md
 To cut a release of Spin, you will need to do the following:
 
 1. Create a pull request that changes the version number for your new version
-   (e.g. 0.1.0 becomes 0.1.1)
-   - `Cargo.toml` is the most important place to make this change
+   (e.g. `0.3.0` becomes `0.3.1`)
+   - Bump the version in Spin's `Cargo.toml`
+   - Bump the version in the Rust SDK as well (`sdk/rust/Cargo.toml`)
+   - Bump the version for the project templates under the `templates` directory
```

Comment:
I assume this means SDK references?  This is fine for Rust but Go is more involved.  For Go, you need to tag the Go SDK, then rev the version in the templates, _then regenerate the `go.sum` file_, and commit that too.

Reply:
Um... I actually don't know... I have been using the archaic process of deleting the sum file, doing a go build, and letting Go recreate it.  But you're right.  There must be a command.